NineChronicles.Headless.Tests/GraphTypes/States/Models/AvatarStateTypeTest.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Nekoyume.Model.State;
+using NineChronicles.Headless.GraphTypes.States;
+using Xunit;
+using static NineChronicles.Headless.Tests.GraphQLTestUtils;
+
+namespace NineChronicles.Headless.Tests.GraphTypes.States.Models
+{
+    public class AvatarStateTypeTest
+    {
+        [Theory]
+        [MemberData(nameof(Members))]
+        public async Task QueryAvatarState(AvatarState avatarState, Dictionary<string, object> expected)
```

Comment:
```suggestion
        public async Task Query(AvatarState avatarState, Dictionary<string, object> expected)
```

Isn't it enough with `Query` as a test name, like you used in `ArenaInfoTypeTest`?